packages/rich-text/src/create.js

```diff
@@ -113,6 +113,7 @@ function toFormat( { type, attributes } ) {
 	};
 }
 
+/* eslint-disable jsdoc/check-line-alignment */
```

Comment:
This is the only case where I disabled the rule because adding an `@example` tag would add a _Usage_ mark in the readme.